plugin.yaml

```diff
@@ -1546,7 +1546,9 @@ node_types:
     interfaces:
       cloudify.interfaces.lifecycle:
         create: azure.cloudify_azure.resources.deployment.create
+        start: azure.cloudify_azure.resources.deployment.pull
```

Comment:
Verify that we can still rerun this over and over again? Also see what happens if we create a new deployment with the same template and execute install and see what happens.